core/src/main/scala/kafka/log/LogConfig.scala

```diff
@@ -66,6 +66,7 @@ object Defaults {
   val MaxIdMapSnapshots = kafka.server.Defaults.MaxIdMapSnapshots
   val MessageDownConversionEnable = kafka.server.Defaults.MessageDownConversionEnable
   val ProducerBatchDecompressionEnable = kafka.server.Defaults.ProducerBatchDecompressionEnable
+  val UnofficialClientLoggingEnable = kafka.server.Defaults.UnofficialClientLoggingEnable
```

Comment:
this config has nothing to do with Log, why is it here? Log here means the log segments stored in kafka broker, not the log4j logging config.